extension/src/toolbar/containers/InitializeApp.tsx

```diff
@@ -22,7 +22,7 @@ export const InitializeApp: FC = () => {
   useEffect(() => {
     const fetch = async () => {
       const templates = await templateClient.findAll();
-      setTemplates(templates);
+      setTemplates(Array.isArray(templates) ? templates : []);
```

Comment:
This is for getting error object when there's no data. Expected value might be `[]`